stat/distmv/normal_test.go

```diff
@@ -572,3 +573,94 @@ func TestNormalScoreInput(t *testing.T) {
 		}
 	}
 }
+
+func TestNormalRandCov(t *testing.T) {
+	const numSamples = 1_000_000
+	const tol = 1e-2
+
+	for cas, test := range []struct {
+		mean []float64
+		cov  []float64
+		semi bool
+	}{
+		{
+			mean: []float64{0, 0},
+			cov: []float64{
+				1, 0,
+				0, 1,
+			},
+		},
+		{
+			mean: []float64{0, 0},
+			cov: []float64{
+				2, 0.9,
+				0.9, 1,
+			},
+		},
+		{
+			mean: []float64{6, 7},
+			cov: []float64{
+				2, 0.9,
+				0.9, 5,
+			},
+		},
+		{
+			mean: []float64{-1, 2, -3},
+			cov: []float64{
+				0, 0, 0,
+				0, 0, 0.1,
+				0, 0.1, 5,
+			},
+			semi: true,
+		},
+	} {
+		for _, covType := range []string{"Cholesky", "PivotedCholesky", "SymDense"} {
+			if covType == "Cholesky" && test.semi {
+				continue
+			}
+
+			name := fmt.Sprintf("case %d with %s", cas, covType)
+
+			n := len(test.mean)
+			a := mat.NewSymDense(n, test.cov)
+			src := rand.NewSource(1)
+
+			var cov mat.Symmetric
+			switch covType {
+			case "Cholesky":
+				var chol mat.Cholesky
+				ok := chol.Factorize(a)
+				if !ok {
+					t.Fatalf("%s: factorization failed, input covariance not positive definite?", name)
+				}
+				cov = &chol
+			case "PivotedCholesky":
+				var chol mat.PivotedCholesky
+				_ = chol.Factorize(a, -1)
```

Comment:
Do we need the `_ = ` prefix?

Reply:
I wanted to leave a hint that there's an ok return but it's not useful here. I've removed the prefix but can put it back if you want.